src/Quasar/Advanced/QuasarAF/Interpreter/Config.purs

```diff
@@ -17,13 +17,8 @@ limitations under the License.
 module Quasar.Advanced.QuasarAF.Interpreter.Config where
 
 import Data.Maybe (Maybe)
-import Network.HTTP.Affjax (URL)
 import OIDC.Crypt.Types (IdToken)
 import Quasar.Advanced.Types (TokenHash)
+import Quasar.QuasarF.Interpreter.Config as QuasarF
 
-type Config r =
-  { basePath ∷ URL
-  , idToken ∷ Maybe IdToken
-  , permissions ∷ Array TokenHash
-  | r
-  }
+type Config r = QuasarF.Config ( idToken ∷ Maybe IdToken, permissions ∷ Array TokenHash | r )
```

Comment:
This is a question, what happened to `basePath`, I saw you put all this into one place from `QuasarAF/Interpreter/Aff.purs` which makes sense but then didn't understand if `basePath` was still needed 😄

Reply:
from `Quasar.QuasarF.Interpreter.Config`:

`type Config r = { basePath ∷ BasePath | r }`

so we're just extending that here, which means we still have the basePath.